pallets/capacity/src/types.rs

```diff
@@ -93,17 +93,32 @@ impl<T: Config> StakingAccountDetails<T> {
 	}
 
 	/// Decrease the amount of active stake by an amount and createa an UnlockChunk.
-	pub fn decrease_by(&mut self, amount: BalanceOf<T>, thaw_at: T::EpochNumber) -> DispatchResult {
-		let new_active = self.active.saturating_sub(amount);
-
-		let unlock_chunk = UnlockChunk { value: amount, thaw_at };
-
-		self.active = new_active;
+	pub fn decrease_by(
```

Comment:
should we actually rename `decrease_by` in StakingAccountDetails and StakingTargetDetails to `deduct` to match the capacity ledger stuff?

Likewise `increase_by` to `deposit`?

Reply:
I like that idea.

Reply:
I like @wilwade 's suggestion of withdraw, partly because it's a much different word.

Reply:
I agree with the use of `withdraw` instead of `deduct`